src/constants.ts

```diff
@@ -29,10 +31,10 @@ for (const fname of fs.readdirSync(dname)) {
   exports.MessageType[name] = value;
 }
 
-exports.NotificationLevel = {
+export const NotificationLevel = {
```

Comment:
Looks like a candidate for `enum`?

Reply:
I thought about it but I wanted to make sure first, that the transpiled code is the same as the original js code.

Reply:
It seems like the generated object is compatible but it contains also the reverted mapping.

```
{
   "0":"LOW",
   "1":"NORMAL",
   "2":"HIGH",
   "LOW":0,
   "NORMAL":1,
   "HIGH":2
}
```

Reply:
Done